risc0/zkvm/src/host/api/convert.rs

```diff
@@ -381,6 +382,32 @@ impl TryFrom<pb::core::SegmentReceipt> for SegmentReceipt {
     }
 }
 
+impl From<Groth16Receipt> for pb::core::Groth16Receipt {
```

Comment:
I don't think we need to worry about Groth16 yet for client/server, we can take a separate pass on that later.

Reply:
Without that we would not be able to cover the case: `InnerReceipt::Groth16` for the `impl From<InnerReceipt> for pb::core::InnerReceipt`.  I'm not sure what's your suggestion here? Remove all the photo stuff related to Groth16? Would that work even if that enum is now part of the InnerReceipt?

Reply:
I would just panic with `unimplemented!` for now and then do a followup PR with just the client/server changes.